crossing-training.lic

```diff
@@ -103,6 +104,8 @@ class CrossingTraining
 
     check_tithe
 
+    check_attunement
```

Comment:
I could add a boolean check. However, I imagine that if you want to train Attunemenet with crossing-training than you wouldn't mind a few second RT every so often. Especially if you wanted to limit the frequency of power walking or leaving the room.